src/lib/components/consignments/setup/info.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import Button from "../../buttons/inverted_button"
+import Welcome from "./welcome"
+
+import {
```

Comment:
iirc it isn't able to catch those, which is why, in reaction-force, I enabled the tsc option to not allow unused vars.